web/src/settings_panel_menu.js

```diff
@@ -50,29 +51,63 @@ export class SettingsPanelMenu {
         this.hash_prefix = opts.hash_prefix;
         this.$curr_li = this.$main_elem.children("li").eq(0);
         this.current_tab = this.$curr_li.data("section");
+        this.current_user_settings_tab = "active-users-admin";
+        this.org_user_settings_toggler = components.toggle({
+            child_wants_focus: true,
+            values: [
+                {label: $t({defaultMessage: "Users"}), key: "active-users-admin"},
+                {
+                    label: $t({defaultMessage: "Deactivated users"}),
+                    key: "deactivated-users-admin",
+                },
+                {label: $t({defaultMessage: "Invitations"}), key: "invites-list-admin"},
+            ],
+            callback: (_name, key) => {
+                browser_history.update(`#organization/user-list-admin/${key}`);
+                this.set_user_settings_tab(key);
+                $(".user-settings-section").hide();
+                $(`[data-user-settings-section="${CSS.escape(key)}"]`).show();
+            },
+        });
 
         this.$main_elem.on("click", "li[data-section]", (e) => {
             const section = $(e.currentTarget).attr("data-section");
 
-            this.activate_section_or_default(section);
-
+            this.activate_section_or_default(section, this.current_user_settings_tab);
             // You generally want to add logic to activate_section,
             // not to this click handler.
 
+            if (section === "user-list-admin") {
+                this.show_org_user_settings_toggler();
```

Comment:
Can this just be called once in `activate_section_or_default` instead of having this here and in `show()` function as well?

Reply:
Yes, fixed [here](https://github.com/zulip/zulip/compare/4e09d5a908b4d7c7c75a93bfcb137d95f6528ff1..881863a3c2531a7c7dd20c4e1fa68bdc28f38e1d)